homeassistant/components/logger/helpers.py

```diff
@@ -68,12 +68,13 @@ def _chattiest_log_level(level1: int, level2: int) -> int:
 
 async def get_integration_loggers(hass: HomeAssistant, domain: str) -> list[str]:
     """Get loggers for an integration."""
-    loggers = [f"homeassistant.components.{domain}"]
+    loggers = {f"homeassistant.components.{domain}"}
```

Comment:
@bdraco looks like you might be familiar with this code - how do you feel about making this hardcoded path the fallback option and relying on `integration.pkg_path` as the default?

Reply:
I think you could drop the hard coding and just use `pkg_path`